tests/cross/src/test/scala/tests/pc/SignatureHelpDocSuite.scala

```diff
@@ -114,17 +106,7 @@ class SignatureHelpDocSuite extends BaseSignatureHelpSuite {
         |""".stripMargin,
     compat = Map(
       "2.12.8" -> foldOlderDocs2,
-      "2.12.7" -> foldOlderDocs2,
-      "2.13" ->
-        s"""|$foldLatestDocs
-            |**Parameters**
-            |- `f`: the function to apply if nonempty.
-            |- `ifEmpty`: the expression to evaluate if empty.
-            |fold[B](ifEmpty: => B)(f: Int => B): B
-            |        ^^^^^^^^^^^^^
-            |  @param ifEmpty String the expression to evaluate if empty.
```

Comment:
The difference was caused by the different implementation of collections, which caused parameters to be returned in a different order.